recon/src/protocol.rs

```diff
@@ -90,6 +90,7 @@ pub async fn initiate_synchronize<S, R, E>(
     recon: R,
     stream: S,
     config: ProtocolConfig,
+    informant: NodeId,
```

Comment:
do we need NodeId here and in the config?

Reply:
nope, removed